style.css

```diff
@@ -0,0 +1,223 @@
+body{
+    font-family: 'Questrial', sans-serif;
+    border: solid black 2px;
+    /*background-color: #BBACC1;*/
+}
+.container {
+    height: 100%;
+    display: grid;
+    grid-gap: 3px;
+    grid-template-columns: 1fr 75% 1fr;
+    grid-template-rows: 80px 250px 200px 400px 225px 600px 400px auto;
+
+}
+
+.container > * {
+    grid-column: 2;
+    min-width: 0;
+}
+
+/**{*/
+    /*border: orangered solid 1px;*/
+/*}*/
+li {
+    list-style: none;
+}
+h2 {
+    text-align: center;
+}
+.head {
+    grid-column: 1 / -1;
+}
+
+.head ul {
+    display: flex;
+
+}
+
+.head ul > li {
+    padding: 30px;
+}
+
+.head ul > li:nth-child(2) {
+    margin-left: auto;
+}
+
+#splash {
+    grid-row: 2;
+    /*grid-column: 2;*/
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-around;
+    /*background-color: greenyellow;*/
+    border-bottom: solid lightcoral 1px;
+}
+
+#splash sub {
+    padding: 2em
+}
+
+#splash a[href] {
+    border: 1px lightgrey solid;
+    border-radius: 5px;
+    padding: .5em;
+}
+
+.quote {
+    font-family: 'Rokkitt', serif;
+    font-style: italic;
+}
+
+.quote blockquote {
+    text-align: left;
+}
+
+.quote p {
+    text-align: right;
+}
+
+#b-quote-1 {
+    grid-row: 3;
+    /*grid-column: 2;*/
+    padding: 50px 110px;
+    border-bottom: solid lightcoral 1px;
+}
+
+.features {
+    grid-row: 4;
+    /*grid-column: 2;*/
+    background-color: greenyellow;
+    border-bottom: solid lightcoral 1px;
+    /*text-align: left;*/
+    /*display: grid;*/
+    /*grid-template-columns: repeat(3, 1fr);*/
+    /*grid-template-rows: 3em 60px 60px;*/
+}
+.features > h2 {
+    text-align: center;
+}
+.features img {
+    float: left;
+}
+
+.features ul{
+    display: flex;
+    flex-wrap: nowrap;
+    justify-content: center;
+    /*align-content: center;*/
+
+}
+/*.f1 strong {*/
+    /*display: inline;*/
+    /*height: 1em;*/
+/*}*/
+
+/*.f1 p {*/
+    /*margin-top: 0;*/
+    /*display: inline;*/
+    /*text-align: left;*/
+/*}*/
+
+
+/*.f1 {*/
+    /*!*vertical-align: middle;*!*/
+    /*display: flex;*/
+    /*border: 1px solid black;*/
+    /*width: 200px;*/
+    /*flex-direction: row;*/
+    /*flex-wrap: wrap;*/
+
+/*}*/
+/*.fi li{*/
+    /*display: flex;*/
+    /*border: gray 1px solid;*/
+/*}*/
+
+#b-quote-2{
+    grid-row: 5;
+    /*grid-column: 2;*/
+    padding: 50px 110px;
+    border-bottom: solid lightcoral 1px;
+}
+
+.team > h2 {
+    text-align: center;
+}
+
+.team {
+    background-color: orangered;
+    grid-row: 6;
+    /*grid-column: 2;*/
+}
+
+/*.team-members li img, span {*/
+    /*display: flex;*/
+    /*width: 100px;*/
+    /*flex-direction: row;*/
+    /*align-items: flex-start;*/
+    /*justify-content: space-evenly;*/
+    /*border-bottom: solid lightcoral 1px;*/
+/*}*/
+
+.team-members li img{
+    border-radius: 50%;
+    -webkit-transition: -webkit-transform .8s ease-in-out;
+    transition:         transform .8s ease-in-out;
```

Comment:
nice use of the transition property! i had seen this around online but don't know how to use it!